inference-engine/src/cldnn_engine/cldnn_engine.cpp

```diff
@@ -163,6 +164,7 @@ InferenceEngine::CNNNetwork clDNNEngine::CloneAndTransformNetwork(const Inferenc
             manager.register_pass<ngraph::pass::BidirectionalLSTMSequenceDecomposition>();
             manager.register_pass<ngraph::pass::BidirectionalGRUSequenceDecomposition>();
             manager.register_pass<ngraph::pass::BidirectionalRNNSequenceDecomposition>();
+            manager.register_pass<ngraph::pass::MVN6Decomposition>();
```

Comment:
MVN6Decomposition is added into default pipeline, so no need to register it here